src/problem2.rb

```diff
@@ -0,0 +1,18 @@
+class Problem2
+  def identify_largest_numbers(array)
+    return [] if array.nil? or array.empty?
```

Comment:
Is an empty array a valid output for this function? By name it has to return largest (2) numbers. In this case it cannot accomplish it's goal because some preconditions are not met.